_includes/navigation.html

```diff
@@ -4,70 +4,70 @@
     {% if site.about_enabled == true %}
       <li>
         <a href="{{ 'about' | relative_url }}" title="About me">
-          <span class="icon icon-about"></span>
+          <svg class="icon-about"><use xlink:href="{% asset_path about.svg %}#icon-about"></use></svg>
```

Comment:
[xlink:href is deprecated according to MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use), and href should be used. Since some browsers may not support it, we should add both properties.